views/partials/navbar.ejs

```diff
@@ -7,21 +7,23 @@
   </a>
 
   <div class="navbar-links-container">
-    <div class="navbar-links"><a href="/gift">🎁 Gift</a></div>
-    <div class="navbar-links"><a href="/app">Get The App</a></div>
+    <div class="navbar-links">
```

Comment:
I remember needing to remove the gift menu item. Did we need to replace it with the Android link though? It seems kind of redundant with the Get Shine Premium link right below it.